borg/archiver.py

```diff
@@ -978,7 +981,46 @@ def do_break_lock(self, args, repository):
             re:^/home/[^/]\.tmp/
             sh:/home/*/.thumbnails
             EOF
-            $ borg create --exclude-from exclude.txt backup /\n\n''')
+            $ borg create --exclude-from exclude.txt backup /
+
+            # exclude the contents of /data/docs/ but not /data/docs/pdf
+            $ borg create -e +/data/docs/pdf -e /data/docs/ backup /
+            # equivalent:
+            $ borg create -e +pm:/data/docs/pdf -e -pm:/data/docs/ backup /
```

Comment:
Ah - this should be removed. (It doesn't work anyways, only `--pattern` supports the new syntax.)